finance-api/user-guides/fetch-purchase-information-for-card-transactions.md

```diff
@@ -0,0 +1,182 @@
+Fetch purchase information for card transactions
+===
+Using the Finance API and the Purchase API, you can fetch purchase information for transactions of payments and refunds that are made with a Zettle card reader. 
+
+* [Prerequisites](#prerequisites)
+* [Step 1: Find the card transaction UUID](#step-1-find-the-card-transaction-uuid)
+* [Step 2: Fetch purchase information for card transactions](#step-2-fetch-purchase-information-for-card-transactions)
+* [Related task](#related-task)
+* [Related API reference](#related-api-reference)
+
+## Prerequisites
+* Make sure that authorization is set up using [Authorization OAuth2 API](../../authorization.adoc). 
+<!-- to be continued if any -->
+
+## Step 1: Find the card transaction UUID
```

Comment:
To be honest i think this is too much. When i read this and try to follow the steps i only get more confused. 
Let's have a meeting  regarding this particular user guide.